<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Z51" failures="1" tests="1" name="Default test" time="7.674" errors="0" timestamp="2021-11-08T12:27:48 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.318">
    <failure type="java.lang.AssertionError" message="User has  registered">
      <![CDATA[java.lang.AssertionError: User has  registered
at org.testng.Assert.fail(Assert.java:99)
at stepdefinitions.register.user_should_get_successfully_registered(register.java:71)
at âœ½.User should get successfully registered(file:///C:/Users/dharshini.4312/git/repository3/AzentioAutomationFramework/AzentioAutomationFramework/src/test/java/features/RegisterAsNewMember.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
